backend/inspirehep/migrator/tasks.py

```diff
@@ -323,6 +325,11 @@ def migrate_and_insert_record(
     )
 
 
+def does_record_have_blacklisted_pid_type(record):
```

Comment:
maybe rename this to ```has_blacklisted_pid_type```?

Reply:
I don't like the name neither but I was trying to make it obvious that you should be reading the function to understand what to pass.

otherwise, how do you know what to pass when you read this function? it could recid, record dict, record class, or only pid_type 😄

Reply:
how about ```is_record_pid_type_blacklisted```?